### PR TITLE
feat: support import cache size by env

### PR DESCRIPTION
**Pull request by su-chang**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -11,10 +11,23 @@ if (logLevel) {
   log.silly('Puppet', 'Config: WECHATY_LOG set level to %s', logLevel)
 }
 
+const CONTACT_CACHE_SIZE = process.env['CONTACT_CACHE_SIZE'] || '3000'
+const FRIENDSHIP_CACHE_SIZE = process.env['FRIENDSHIP_CACHE_SIZE'] || '20'
+const MESSAGE_CACHE_SIZE = process.env['MESSAGE_CACHE_SIZE'] || '50'
+const ROOM_CACHE_SIZE = process.env['ROOM_CACHE_SIZE'] || '50'
+const ROOM_INVITATION_CACHE_SIZE = process.env['ROOM_INVITATION_CACHE_SIZE'] || '20'
+const ROOM_MEMBER_CACHE_SIZE = process.env['ROOM_MEMBER_CACHE_SIZE'] || '2000'
```
Comment (windmemory) [inline src/config.ts:19]: It would be better to not change the default value of the current cache.

**Review by windmemory — CHANGES_REQUESTED** [review]

**Review by windmemory — APPROVED** [review]
Looks good to me, approved!

**Review by huan — CHANGES_REQUESTED** [review]
Thanks for the improvements!

Please use a formated env var name with `WECHATY_PUPPET_` namespace:

  CONTACT_CACHE_SIZE -> WECHATY_PUPPET_LRU_CACHE_SIZE_CONTACT
  FRIENDSHIP_CACHE_SIZE -> WECHATY_PUPPET_LRU_CACHE_SIZE_FRIENDSHIP
  MESSAGE_CACHE_SIZE -> WECHATY_PUPPET_LRU_CACHE_SIZE_MESSAGE
  ROOM_CACHE_SIZE -> WECHATY_PUPPET_LRU_CACHE_SIZE_ROOM
  ROOM_INVITATION_CACHE_SIZE -> WECHATY_PUPPET_LRU_CACHE_SIZE_ROOM_INVITATION
  ROOM_MEMBER_CACHE_SIZE -> WECHATY_PUPPET_LRU_CACHE_SIZE_ROOM_MEMBER

**Review by windmemory — APPROVED** [review]

**Review by huan — APPROVED** [review]
LGTM

Thank you very much for the improvements!

**Pull request merged**